libraries/botbuilder-dialogs-adaptive/package-lock.json

```diff
@@ -0,0 +1,764 @@
+{
```

Comment:
Please check whether these changes are intended.

Reply:
I deleted it, ran all the tests and they all passed correctly. Therefore this change isn't required.